.github/pull_request_template.md

```diff
@@ -1,12 +1,21 @@
 #### Trac ticket number
-<!-- Replace XXXXX with the corresponding Trac ticket number, or delete the line and write "N/A" if this is a trivial PR. -->
+<!-- Replace XXXXX with the corresponding Trac ticket number. -->
+<!-- Or delete the line and write "N/A" if this is a trivial PR. -->
 
 ticket-XXXXX
 
 #### Branch description
 Provide a concise overview of the issue or rationale behind the proposed changes.
 
+#### AI Assistance Disclosure (REQUIRED)
+<!-- Please select exactly ONE of the following: -->
+- [ ] **No AI tools were used** in preparing this PR.
+- [ ] **AI tools were used**, and I have disclosed which ones, and fully reviewed and verified their output.
+
 #### Checklist
+<!-- Contributing guidelines: https://docs.djangoproject.com/en/stable/internals/contributing/writing-code/submitting-patches/ -->
+<!-- Security policies: https://docs.djangoproject.com/en/stable/internals/security/ -->
+- [ ] This PR follows the contribution guidelines and security reporting policies.
```

Comment:
I'm not sure if this is to discourage security disclosure via PRs? If so I think we should have something more aggressive and bold, something like:
```
- [ ] **This DOES NOT disclose a security vulnerability** (these should be reported to security@djangoproject.com).
```